lib/spark_api/models/listing.rb

```diff
@@ -272,6 +292,10 @@ def setup_attribute(attributes, collection, klass)
         end
       end
 
+      def subresource_path(subresource)
+        return "#{self.class.path}/#{self.Id}/#{subresource}/"
```

Comment:
Getting into nitpick territory here:

- Ruby automatically returns the value of the last statement in a method, so the `return` here is optional, and I _think_ the idiomatic approach would be to omit it.
- I can't tell if this is just a style thing or if it actually makes a difference. I'll mention it, but you can feel free to tell me to shove off. The trailing `/` here feels out of place as it's not really part of the subresource path and it changes the route called in line 191 (though probably not meaningfully).